src/main/java/pl/touk/sputnik/connector/stash/StashFacade.java

```diff
@@ -56,6 +56,16 @@ public Connectors name() {
         }
     }
 
+    private String getFilePath(ReviewElement container) {
+        String filePath;
+        if (container.parent.isEmpty()) {
+            filePath = container.name;
+        } else {
+            filePath = String.format("%s/%s", container.parent, container.name);
```

Comment:
container.toString seems to contain concatenation of container.parent and container.name, but I don't know why it wasn't used before.